specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -124,9 +124,9 @@ function ConditionalFormatter({
 
   return (
     <div
-      className={`flex
-        ${isExpanded || !hasCondition ? 'flex-col' : 'items-center'}
-        ${isExpanded ? 'gap-2' : ''}
+      className={`flex gap-2
+        ${isExpanded || !hasCondition ? 'flex-col' : ' items-center pb-2 pt-2'}
+        ${isExpanded ? 'gap-2 pt-2' : ''}
```

Comment:
gap-2 is not necessary in line 129, you are applying it twice. 
Could you also move the pt-2 to line 127 ?
If yes it would remove the need of line 129 and the need of pt-2 in line 128.